css-color-4/Overview.bs

```diff
@@ -4714,7 +4765,7 @@ Color Space for Interpolation</h3>
 	Thus, different color spaces may be more appropriate for each interpolation use case.
 
 	* In some cases, the result of physically mixing two colored lights is desired.
-		In that case, the CIE XYZ  or srgb-linear color space is appropriate, because they are linear in light intensity.
+		In that case, the CIE XYZ, display-p3-linear  or srgb-linear color spaces are appropriate, because they are linear in light intensity.
```

Comment:
Should these be linked to their definition?

Reply:
Agreed